src/App.jsx

```diff
@@ -14,70 +15,81 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {!value.length
+        ? (
+          <h1 className="title is-flex is-align-items-center">
+            No goods selected
+          </h1>
+        )
+        : (
+          <h1 className="title is-flex is-align-items-center">
+            {value}
+            {' '}
+            is selected
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
               type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+              className="delete ml-3"
+              onClick={() => {
+                setValue('');
+              }}
+            />
+          </h1>
+        )
+}
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map((good) => {
+            const isGoods = x => x === value;
```

Comment:
```suggestion
            const isSelected = x => x === value;
```
also use more meaningful variable names